pytrafikverket/trafikverket.py

```diff
@@ -106,8 +106,8 @@ def generate_node(self, parent_node):
 class Trafikverket(object):
     """Class used to communicate with trafikverket api."""
 
-    _api_url = "http://api.trafikinfo.trafikverket.se/v1.2/data.xml"
-    date_time_format = "%Y-%m-%dT%H:%M:%S"
+    _api_url = "https://api.trafikinfo.trafikverket.se/v2/data.xml"
+    date_time_format = "%Y-%m-%dT%H:%M:%S.%f%z"
```

Comment:
It says on the API website that only modified time has the additional "z" as it's UTC while all other times are Europe/Stockholm so why are we changing this?